pkgs/os-specific/linux/kernel/grsecurity-config.nix

```diff
@@ -0,0 +1,117 @@
+{ stdenv, version }:
+
+with stdenv.lib;
+
+''
+GCC_PLUGINS y
+
+# Auto configuration with these constraints will enable most of the
```

Comment:
This comment no longer applies. The reason I originally did not list out every config is that it is verbose and makes it hard to tell how this config deviates from what upstream recommends. Comments like "something about symlinks" could be construed as toggling things without due consideration.